lib/fathead/django_built_ins/parse.py

```diff
@@ -0,0 +1,177 @@
+from bs4 import BeautifulSoup
+
+DJANGO_HOME = 'https://www.djangoproject.com/'
+DJANGO_DOC_URL = 'https://docs.djangoproject.com/en/1.9/ref/templates/builtins/'
```

Comment:
I would prefer to use just one file for configuration purposes, in this case, you're only specifying the version, but maybe in the future, you could use it to control other common attributes. Please let me know what do you think.